src/containers/audio-rewards-page/components/ExplainerTile.module.css

```diff
@@ -60,6 +54,7 @@
   font-size: 40px;
   line-height: 49px;
   background: var(--page-header-gradient);
+  background-clip: text;
```

Comment:
linter complained about not having the non-webkit version of this for compatibility